common/app/com/commercetools/sunrise/common/sessions/SessionHandlerBase.java

```diff
@@ -0,0 +1,69 @@
+package com.commercetools.sunrise.common.sessions;
+
+import io.sphere.sdk.json.JsonException;
+import io.sphere.sdk.json.SphereJsonUtils;
+import org.slf4j.Logger;
+import play.libs.Json;
+import play.mvc.Http;
+
+import javax.annotation.Nullable;
+import java.util.List;
+import java.util.Optional;
+
+public abstract class SessionHandlerBase<T> implements SessionHandler<T> {
```

Comment:
Is handler the right concept? I am not sure if it is some kind of registry. http://martinfowler.com/eaaCatalog/registry.html